sites/partners/page_content/locale_overrides/general.json

```diff
@@ -8,5 +8,11 @@
     "header": "Detroit Housing Portal is a project of the",
     "headerUrl": "https://detroitmi.gov/departments/housing-and-revitalization-department",
     "headerLink": "City of Detroit - Housing and Revitalization Department"
+  },
+
+  "application": {
+    "details": {
+      "countyName": "Alameda County"
```

Comment:
nit: I'm not sure where this "countyName" is used, but maybe we can omit this whole "application" section?

Reply:
Done